src/ClientProtos/capture_data.proto

```diff
@@ -170,6 +171,7 @@ message TimerInfo {
   uint64 timeline_hash = 11;
   repeated uint64 registers = 12;
   Color color = 13;
+  string cgroup_name = 14;
```

Comment:
I can't really judge if this could have any performance implications. I guess not, but someone else should have a look - in particular at this change to the TimeInfo proto.

Reply:
@dimitry- @dpallotti Can you take a look if this is alright? 

Other strings, like e.g. the timeline string, are stored elsewhere and the TimerInfo just has a string hash. I don't have the full context if we should do the same here. I'm also not sure if we have changed our string handling in any way.

Reply:
Yeah, this probably duplicates the string for every "timer" and hence has memory implications. I guess a key should be used like some fields to above. 

The good thing is that we don't use `TimerInfo` for serialization anymore (only for deserialization from the old format).

Reply:
You can have a look at `GpuDebugMarkerTrack` and `GpuSubmissionTrack` and what they do with `App::string_manager_`.

Reply:
Thanks for the suggestion. Now I change to store the string hash in the `TimerInfo` and recover the string value with `App::string_manager_`.

Also the `OnMemoryUsageEvents` method, which converts a  `MemoryUsageEvent` into a `TimerInfo`, has been moved from `App.cpp` to `CaptureEventProcessor.cpp`. And the corresponding test has been added to `CaptureEventProcessorTest.cpp`.